### ISSUE TITLE
Hide cancer hallmarks table when API does not return hallmarks

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#947]
For targets like [TMPRSS2](https://www.targetvalidation.org/target/ENSG00000184012), the Cancer Hallmarks section shows information about the gene function and an empty table.

![Screenshot 2020-03-27 at 17 38 06](https://user-images.githubusercontent.com/7490258/77784020-bdf78a80-7051-11ea-8728-4da83a73ae70.png)

In the [cancer-hallmark plugin code](https://github.com/opentargets/webapp/tree/master/app/plugins/cancer-hallmark), the Angular app will display the table based if the API response has `target.hallmarks`. 

@LucaFumis, if we made the following changes to the [cancer-hallmark.html](https://github.com/opentargets/webapp/blob/master/app/plugins/cancer-hallmark/cancer-hallmark.html) file, would that resolve the issue?

1. Update line 1 to `<div ng-if="target.hallmarks.cancer_hallmarks">`
2. Update line 105 to `<div ng-if="!target.hallmarks.cancer_hallmarks"><p>No data available</p></div>`

I propose those changes because the API response for hallmarks looks like the following:

```
"hallmarks": {
  "cancer_hallmarks": [ ... ],
  "attributes": [ ... ],
  "function_summary": [ ... ]
},
```

If we look specifically for `hallmarks.cancer_hallmarks`, that should ensure the section appears only for targets with cancer hallmarks?

**Comment by andrewhercules** [comment]
Issue closed as work completed as part of 20.04 release process

**Comment by andrewhercules** [comment]
Issue closed as work completed as part of 20.04 release process
